frontend/src/app/AdminNavBar/AdminNavBar.tsx

```diff
@@ -80,11 +85,47 @@ export interface AdminNavBarProps {
 export const AdminNavBar = ({ isMenuOpen }: AdminNavBarProps): JSX.Element => {
   const { user } = useUser()
 
+  const ROLLOUT_ANNOUNCEMENT_KEY = useMemo(
+    () => ROLLOUT_ANNOUNCEMENT_KEY_PREFIX + user?._id,
+    [user],
+  )
+  const [hasSeenAnnouncement] = useLocalStorage<boolean>(
+    ROLLOUT_ANNOUNCEMENT_KEY,
+  )
+
+  // Only want to show the modal if user id exists but user has no emergency contact
+  const shouldShowModal = user && !!user._id !== !!user.contact
```

Comment:
The inequality check with the double negation looks a bit funny based on the comment:
```suggestion
const shouldShowModal = user && user._id && !user.contact
```

As a side note: it's not necessary to double negate both side of a equality/inequality check, one time is enough 😅 
```javascript
!user._id !== !user.contact
```

Reply:
oh yeah thanks tim! forgot to change it to `&&` after i'd added `user &&` to the inequality check